releases/release-5.0.3.md

```diff
@@ -0,0 +1,115 @@
+    - Fix the issue that Backup & Restore reports the error of "file already exists" when TDE is enabled during the restore [#10421](https://github.com/tikv/tikv/pull/10421)
+    - Skip clearing callback during graceful shutdown to avoid breaking ACID in some cases [#10396](https://github.com/tikv/tikv/pull/10396)
+    - Fix a bug that the read index is shared for replica reads on a Leader [#10391](https://github.com/tikv/tikv/pull/10391)
+    - Fix the wrong function that casts `DOUBLE` to `DOUBLE` [#10388](https://github.com/tikv/tikv/pull/10388)
+
++ TiFlash
+
+    - Fix the issue that TiFlash keeps restarting because of the split failure
+    ?- Fix the potential issue that TiFlash cannot GC delta data
+    - Fix a bug that TiFlash adds wrong padding for non-binary characters in the `CAST` function
+    - Fix the issue of incorrect results when handling aggregation queries with complex `GROUP BY` columns
+    - Fix the TiFlash panic issue that occurs under heavy write pressure
+    - Fix the panic that occurs when the right jon key is not nullalbe and the left join key is nullable
+    - Fix the potential issue that the `read-index` requests take a long time
+
++ Tools
+
+    - BR
+
+        * Fix parquet parse when parse decimal type [#1277](https://github.com/pingcap/br/pull/1277)
+        * Fix the bug that lightning returns EOF error when CSV file without '\r\n' at the last line and `strict-format = true` [#1189](https://github.com/pingcap/br/pull/1189)
+        * Fix the bug that lightning rebase wrong auto_increment base when the auto_increment field type is float or double [#1186](https://github.com/pingcap/br/pull/1186)
+
+## 以下 note 未分类。请将以下 note 进行分类 (New feature, Improvements, Bug fixes, Compatibility Changes 四类)，并移动到上面对应的标题下。如果某条 note 为多余的，请删除。如果漏抓取了 note，请手动补充
+
++ PD
+
+    - release-note [#3798](https://github.com/pingcap/pd/pull/3798)
+
++ Tools
+
+    - TiCDC
+
+        * Fix potential deadlocks [#2142](https://github.com/pingcap/ticdc/pull/2142)
+        * Fix a bug that could cause cdc server panic because of the late calculation of resolved ts [#2047](https://github.com/pingcap/ticdc/pull/2047)
+        * Fix panic when TiCDC fails to read `/proc/meminfo` [#2024](https://github.com/pingcap/ticdc/pull/2024)
+        * Reduce unnecessary memory consumption [#2012](https://github.com/pingcap/ticdc/pull/2012)
+        * Make sorter IO errors more user-friendly. [#1977](https://github.com/pingcap/ticdc/pull/1977)
+        * Fix Unified Sorter memory consumption when tables are many. [#1958](https://github.com/pingcap/ticdc/pull/1958)
+        * Decrease the default gRPC connection pool size to decrease goroutines count. [#1951](https://github.com/pingcap/ticdc/pull/1951)
+        * Fix a bug that some MySQL connection could leak after MySQL sink meets error and pauses. [#1946](https://github.com/pingcap/ticdc/pull/1946)
```

Comment:
```suggestion
    - TiCDC

        * Fix the problem that cdc owner exited due to emit checkpoint failure [#2031] (https://github.com/pingcap/ticdc/pull/2031)
        * Fix a bug that some MySQL connection could leak after MySQL sink meets error and pauses [#1946](https://github.com/pingcap/ticdc/pull/1946)
        * Fix panic when TiCDC fails to read `/proc/meminfo` [#2024](https://github.com/pingcap/ticdc/pull/2024)
        * Reduce memory consumption [#2012](https://github.com/pingcap/ticdc/pull/2012), [#1958](https://github.com/pingcap/ticdc/pull/1958)
        * Fix a bug that could cause cdc server panic because of the late calculation of resolved ts [#2047](https://github.com/pingcap/ticdc/pull/2047)
        * Fix potential deadlocks for processor [#2142](https://github.com/pingcap/ticdc/pull/2142)

    - Backup & Restore (BR)

        * Fix the bug that all system tables are filtered when restoring [#1224](https://github.com/pingcap/br/pull/1224)

    - TiDB Lightning

        * Fix lightning panic problem for some special data [#1268](https://github.com/pingcap/br/pull/1268)
        * Fix lightning split large csv file EOF error during the split progress [#1189](https://github.com/pingcap/br/pull/1189)
        * Fix the bug that lightning rebase wrong auto_increment base when the auto_increment field type is float or double [#1186](https://github.com/pingcap/br/pull/1186)
        * Fix parquet parse for decimal types [#1277](https://github.com/pingcap/br/pull/1277)
```